testbed/tests/widgets/test_splitcontainer.py

```diff
@@ -55,7 +56,7 @@ async def content3_probe(content3):
 
 @pytest.fixture
 async def widget(content1, content2):
-    xfail_on_platforms("android", "iOS")
```

Comment:
I marked this as xfail because I can't think of any meaningful analog of a SplitContainer view on mobile, so SplitContainer is "not implemented on purpose", rather than "not implemented yet". Have you got something in mind for SplitContainer?

Reply:
OK, that makes sense. I've changed it back to `xfail` and added some comments to explain the difference.